vault/resource_pki_secret_backend_config_cmpv2.go

```diff
@@ -118,14 +118,8 @@ func pkiSecretBackendConfigCMPV2Write(ctx context.Context, d *schema.ResourceDat
 		}
 	}
 
-	if authenticatorsRaw, ok := d.GetOk(consts.FieldAuthenticators); ok {
-		authenticators := authenticatorsRaw.([]interface{})
-		var authenticator interface{}
-		if len(authenticators) > 0 {
-			authenticator = authenticators[0]
-		}
-
-		data[consts.FieldAuthenticators] = authenticator
+	if authenticators, ok := getListOfNotEmptyMaps(d, consts.FieldAuthenticators); ok {
+		data[consts.FieldAuthenticators] = authenticators
```

Comment:
Will these changes now cause a difference on existing state files?

Reply:
I don't think so? But I am not sure, I'll test.

Reply:
It does not.